src/ui/controls/edit.cpp

```diff
@@ -84,7 +84,17 @@ bool IsSep(int character)
     return !IsWord(character);
 }
 
+bool IsBreaker(char c)
+{
+    return ( c == '.'  || c == '{' || c == '}' ||
+             c == ';' || c == ':' || c == '[' || c == ']' ||
+             c == '(' || c == ')' || c == '=' );
```

Comment:
Well I would add also +-*\%,? but I`am not sure how Colobot Team want to see it.